### ISSUE TITLE
Studying "What is Really Happening at Your Dental Cleaning?"

### ISSUE THREAD

**Issue by hyunsik-yoon** [labsurde-life/english#10]
https://www.youtube.com/watch?v=uTiC467dwUM

**Comment by hyunsik-yoon** [comment]
## part 1
https://youtu.be/uTiC467dwUM

Today I did something very exciting. I brought my camera to work.

So my coworker _slash friend_ she was very nice and let us record her clinic applointment.

- 말 중간에 slash 를 쓰기도 한다. : my coworker _slash friend_

I am very excited to show and explain all the _ins and outs of_ what ~go under~ _goes on during_ dental cleaning routine every six months.

- ins and outs: The particular details and nuances of a situation, task, etc.
  - "I'm not surprised he made such a stupid mistake—he doesn’t know the ins and outs of this business."
- ins and outs (of ~):  the special things that one needs to know to do ~
  - "Jane knows the ins and outs of repairing computers."
  - 
Without further ~redue~ _ado,_ here's what to expect at a routine cleaning appointment.

- without further ado: without waiting any longer
  - "Without further ado, I'd like to introduce our speaker."

First, you'll alomost likely be greeted in the waiting room. The hygienist usually call your name and bring you back to the room. 

- hygienist : 위생사
  - "dental hygenist" 

Next, they will usually place _bib_ on you and ask you how's the day going, some smalltalk, how's the weather, things like that.

- bib: 턱받이

They will go over any changes in your medical history, any new medication, things like that. They will also ask how your teeth are feeling, if anything's bothering you or  _if_ you just here for routine cleaning or exam.

- go over: consider, examine, or check something:
  - "I want to go over these plans with you again"

**Comment by hyunsik-yoon** [comment]
### Part 2
https://youtu.be/uTiC467dwUM?t=70

Next they will let you know if you are ~do~ _due_ for an x rays today.

Bitewing X rays, you, generally taking about once a year. here's the sensor used by digital x rays

- bitewing
<img width="257" alt="image" src="https://user-images.githubusercontent.com/63768207/212502889-8fcedb21-d6a2-4a94-9ca7-f41aca54f2e9.png">

- crown
금니 등을 할 떄 이빨에 씌우는 것. 재료로는 금속, 세라믹, 등등이 있음

Bitewing x rays are just two picture each side of your your mouth.

and they specifically check for cavaties, tar, and the level of the bone.

- cavaty 충치. an area of decay in a tooth
  - She recently _developed_ a _cavity_ in one of her lower back teeth.  그녀는 최근에 아래 어금니에 충치가 생겼다. 

sometimes if you have high cavaty rate, or your bone is monitored, these may be taken at every cleaning appointment.

Here you can see a little closer up on how we put those in your mouth

I know that they are not the most comfortable things...
but oh. look at that good picture. that was a good _premolar_ shot.

- molar, back tooth  어금니
- front teeth 앞쪽 이빨
- grit: To clamp (the teeth) together.
  - She gritted her teeth against the pain.  그녀는 이를 악물고 통증을 참았다. 
- c'anine / canine tooth  / eyetooth 송곳니
- wisdom teeth 사랑니
  -  I had my wisdom tooth pulled out. 또는 I had my wisdom teeth taken out. 나 사랑니 뽑았어.

**Comment by hyunsik-yoon** [comment]
### Part 3.
https://youtu.be/uTiC467dwUM?t=134

Every 3 to 5 years, they may take routine full mouth x rays, fmx. There ones are checking the roots of every single tooth, all of the front teeth, and all of the back teeth, about 18 images are taken in a ~form of~ full mouth x ray.

you can _kinda_ see here how you can see the roots of tooth inside the bone there.

it is possible that your dentist will want to take a panoramic x ray which shown here at the 3 to 5 year marks. These are generally taken to exam wisdom teeth, or jar, tmj or other surrounding structure.

- TMJ disorder : 턱관절 장애

next, before your cleaning, we need to check your gums about once a year. 

We use a little ruler called perio ~pro~ _probe_ which takes measurements of your gums in millimeter. 

- **take measurements**: to measure something in order to know its exact size with a measuring tape or gadget.
  - "This dental probe takes measurements of your gums in millimeter."
- **take measures**: plan and take action; take steps to do something
  - "The government needs to take measures to deal with the homeless situation."
 
Basically, there's a natural space between your gums and tooth. We measure in that space to chech the height of the bone in the surrounding tissues.

**Comment by hyunsik-yoon** [comment]
### Part 4. 
https://youtu.be/uTiC467dwUM?t=184

healthy tissue is between 1 and 2 millimeters.
so, 1s, 2s, or 3s are considered within normal limit. 
4s are considered border line. possibly and area of information of tar ?

- borderline 경계선
- tartar 치석. (발음이 참.... [타r러r] 처럼 들림)

the number 5 or up often indicates ?? ~bone loss~ or gum disease in the area.

we check this number once a year to confirm your gums are healty or happy. 

as you can see here, often times these numbers are recorded in your chart for comparison thought the year.

- often 이 oftentimes 보다 덜 구식처럼 들림
- oftentimes: = often. [oftentimes 는 옛날말처럼 들림](https://writingexplained.org/often-times-or-oftentimes-difference)

The black numbers 1, 2 and 3 are healthy and the red numbers 4 and up show areas to be monitored, things like that.

One more thing before we jump into the cleaning, oftentime the hygieninst and _and/or dentist_ do an oral cancer screening.

- and/or dentist 발음이....... 앤도r대니스트.. 너무 자연스럽게 and ordentist 처럼 들림.

basically they will check the ~latteral~ _lateral_ sides of the tongue, underneath the tougue, inside of your cheeks, inner lips, back of the pallet and throat for any suspicious lumps or bumps

- lateral: of or pertaining to the side (측면의), of outer side of the body part
- palate [pa-lət] : the roof of the mouth. 입천정인듯.
- lumps vs bumps
  - lumps: 서로 들러붙어있는 무언가 (?)
    - ex) Stir the gravy until there are no more lumps .
    - “When patients complain about lumps, it usually refers to something on the skin that feels elevated or something under the skin that elevates it,” says Lucia Seminario-Vidal, MD, PhD

- lump 와 bump의 차이점이 뭔지..?
- lump and bump 라고 같이 쓰기도 하는듯
- 
now it's time to get the tray ready for your cleaning. The reason you're here, you want clean and shiny teeth.

**Comment by hyunsik-yoon** [comment]
### part 5
https://youtu.be/uTiC467dwUM?t=241

first, a lot of time we start with an ultrasonic scaler, either called pazon or cabitron.

we clean around each tooth with this instrument first, which takes off most of tartar, stein, and plaque very well. 

The combination of untranosic vibration on this instrument and fiushing of water cleans teeth very well.

The ~sucktion~ _suction_ will be in your mouth whole time. It's a lot of messy water.

- suction: 흡입. (철자 유의)

The hygienist may blow some air in your teeth and then start scaling with a hand insturment, the instruments that make the ~scruping~ scraping sound.

- hand insturment
  - 손을 쓰는 기기들인데.... 주로 치과나 음악에서 많이 뜨는 용어인 듯.
  - 음악에서는 탬버린, 드럼, 북 같은 percussion 종류를 주로 말하는 듯.
- scraping sound, scraping noise
  - 날카로운 것이 긇히면서 나는 소음
- scratching sound
  - sound of someting being scratched
    - ex) "We heard further scratchings at the door as the dog whined to be let in."

_The drier_ the tooth, _the easier it is_ to see any residual tartar upon the surfcae.

After the teeth are smoothen out with the scalers, it is time to polish.

There are two ways to polish. One way is with polishing cup a paste as seen here. Today out patient choose strawberry falvoured paste. 

Basically the goal of polishing is to remove ~stein~ _stain_ and smoothen the tooth surface so plaque is less likely to attach.

**Comment by hyunsik-yoon** [comment]
### Part 6
From https://youtu.be/uTiC467dwUM?t=300

now, just for the sake of this video, i am pretending to air polish.

this is the other to polish teeth. I don't use this often, it's very salty and messy for me but some people love it.

we're just pretending here because if we were really air polishing, she would be wearing ~protected~ _protective_ goggles I wear.

 _On rare occasions I do use this_ but most of time I use cup and paste _where you get to pick your_ flavor and everyone is happy.

- on occasion: from time to time, frequently
  - ex) "He lives in the country, though he visits the city on occasion."
- on rare occasions: infrequently
  - ex) "On rare occasions you can see this, a ring stretching around the entire sky."
 
lastly, we floss. Floss all the ~context~ _contacts_ very well. We use correct technique. you can see my fuzzing video on how to floss correctly.

Before you leave, most kids under the age of sometimes 19, 22 wat we have you
most yonger people will receive ~flo??~ _floride_ before you leave.

There're generally two options. the ~floride trades~ _florite rays_, which we're showing now, they're great but you cannot eat or drink for 30 minutes after. 

The florite ??, which is shown now, is great because you don't have to wait until you drink, The only downside to this is that your teeth sometime feel sticky and bumpy for a while. 

- downside: negative aspect

ok. now we're done with your cleaning. after your exam with your doctor, you'll schedule your next cleaning appointment. get a good?? _goodie bag_ and enjoy the rest of day.

**Comment by hyunsik-yoon** [comment]
### Questions
1. Have you had any experience with cleaning teeth in dental clinic? Could you describe your experience?
2. Do you have good memory with a dentist? What was that?
3. What are the procedure of cleaning teeth?
4. Do you remeber x rays in dental clinic shown in the video? Can you explain what are those?
  -  bitewing x ray : to check cavaty, tartar, etc
  - full mouth x ray (FMX): to check root of teeth
  - panoramic x ray: to examin wisdom teeth, jaw, tmj, and surrounding structures
5. Do you remember how dentists check oral cancer?
6. If you see a dentist someday, is there a question you'd like to ask?
